static/js/task/controllers/task.controller.js

```diff
@@ -22,17 +22,22 @@
             self.task_worker_id = $routeParams.taskWorkerId;
             self.task_id = $routeParams.taskId;
 
-            if (!self.task_worker_id) { //if they navigate away mid-queue and then attempt task from task feed
```

Comment:
see deleted comment. this now breaks if you stop mid-queue and then go back to task feed and attempt the same task.